api/tests/identity-access-management/integration/domain/services/refresh-token-service.test.js

```diff
@@ -34,6 +35,90 @@ describe('Integration | Identity Access Management | Domain | Service | refresh-
         const refreshTokensInDb = await refreshTokenService.findByUserId(userId);
         expect(refreshTokensInDb).to.deep.equal(['123:pix-orga:XXX-123-456']);
       });
+      context('set refresh token expiration by scope', function () {
+        let userId, source, uuidGenerator;
+        let refreshTokenTemporaryStorage, userRefreshTokensTemporaryStorage;
+
+        beforeEach(async function () {
+          userId = '123';
+          source = 'APP';
+          uuidGenerator = () => 'XXX-123-456';
+
+          refreshTokenTemporaryStorage = {
+            save: sinon.stub(),
+          };
+
+          userRefreshTokensTemporaryStorage = {
+            lpush: sinon.stub(),
+            expire: sinon.stub(),
+          };
+        });
+
+        it('sets a default refresh token lifespan when scope is not defined', async function () {
```

Comment:
Petite suggestion de formulation : 
```suggestion
        it('sets a default refresh token lifespan when scope is not given', async function () {
```